### DISCUSSION TITLE
bootstrap-vue <b-form-datepicker> custom date format

### DISCUSSION THREAD

**Discussion by edijsci** [bootstrap-vue/bootstrap-vue#6515] [Q&A]
Hi! 
I use  `<b-form-datepicker>`  form in my project
with `:date-format-options="{
                                    year: 'numeric',
                                    month: 'numeric',
                                    day: 'numeric'
                                }"`.       
Date field displays format in yyyy.MM.dd, is there possible to change date format to dd.MM.yyyy?

**Comment by xjimdim** [comment]
Can somebody answer that? 
I cant find a way to show date as dd.MM.yyyy
